src/topology/algebra/ordered.lean

```diff
@@ -282,6 +282,14 @@ is_open_Iio.interior_eq
 @[simp] lemma interior_Ioo : interior (Ioo a b) = Ioo a b :=
 is_open_Ioo.interior_eq
 
+lemma eventually_le_of_tendsto_lt {l : filter γ} {f : γ → α} {u v : α} (hv : v < u) :
+  tendsto f l (𝓝 v) → ∀ᶠ a in l, f a ≤ u :=
+λ h, eventually.mono (tendsto_nhds.1 h (< u) is_open_Iio hv) (λ v, le_of_lt)
```

Comment:
Two separate small points, for this lemma and the next:
- all hypotheses go before the `:`, since it saves a leading `λ h, `:  so here, `(h : tendsto f l (𝓝 v))` : ....`
- does the proof term work using projection notation? `(tendsto_nhds.1 h (< u) is_open_Iio hv).mono`